src/current/_includes/releases/v22.2/v22.2.17.md

```diff
@@ -0,0 +1,33 @@
+## v22.2.17
+
+<!-- Release Date: November XX, 2023 -->
+
+{{ site.data.alerts.callout_info }}
+v22.2.17 is currently available only on select CockroachDB {{ site.data.products.dedicated }} clusters. If you wish to upgrade your {{ site.data.products.dedicated }} cluster to v22.2.17, please submit a [support request](https://support.cockroachlabs.com/hc/en-us/requests/new).
+
+This release is currently not available for CockroachDB {{ site.data.products.serverless }} or {{ site.data.products.core }} customers.
+{{ site.data.alerts.end }}
+
+<!-- {% include releases/release-downloads-docker-image.md release=include.release %} -->
+
+<h3 id="v22-2-17-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that could prevent RESTORE from working if it was performed during a cluster upgrade. [#112757][#112757]
+- Queries with the `st_union` aggregate function could produce incorrect results in some cases due to the query optimizer performing invalid optimizations. This bug has been present since the `st_union` function was introduced in version 20.2.0. [#112796][#112796]
```

Comment:
Link built-in functions doc.

Reply:
Added the link!  Also linked to the optimizer doc here.